bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue

```diff
@@ -544,6 +544,10 @@ export default {
           path: 'zwave-network',
           popup
         }
+      }, {
+        props: {
```

Comment:
So the `bridgeUID` is passed to the popup in this lines and doesn't need to be passed in the template here:

https://github.com/ghys/openhab-webui/blob/cbfc0e83c9c89e62e1c288a35426d8c632632cd4/bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue#L126

Reply:
Yep, I think you're right...

https://github.com/ghys/openhab-webui/blob/cbfc0e83c9c89e62e1c288a35426d8c632632cd4/bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue#L126

seems like a leftover from a previous iteration, as is

https://github.com/ghys/openhab-webui/blob/cbfc0e83c9c89e62e1c288a35426d8c632632cd4/bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue#L296

The change was made a while ago to make on-the-fly "routes" for those popups, the goal was to allow "going back" with standard browser controls (when for instance you have a "back" button on your mouse or a gesture to go back on your touch screen, this would work as intended instead of the page "behind" going back).

Reply:
Okay, thanks for the explanation. 

So https://github.com/ghys/openhab-webui/blob/cbfc0e83c9c89e62e1c288a35426d8c632632cd4/bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue#L126 can be removed, right?

Reply:
Absolutely.